sgt-back-end/index.js

```diff
@@ -0,0 +1,119 @@
+const pg = require('pg');
+const port = 3000;
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+const express = require('express');
+const app = express();
+const ok = require('./queries');
```

Comment:
I like putting these queries all in one place!
I probably wouldn't name this `ok` though, since that isn't very descriptive.